edx_proctoring/templates/proctored_exam/failed-prerequisites.html

```diff
@@ -0,0 +1,48 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam entrance" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      This exam is proctored.
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+    This exam is proctored but you have not satisfied the prerequisites for this exam. All requirements for this exam must be satisfactory before you can proceed to take this proctored exam in order to continue to be eligible for credit. You can see the requirements (in order) listed on your <a href="{{progress_page_url}}">Progress</a> page.
```

Comment:
"You have not satisfied the prerequisites for this exam. All requirements must be satisfied before you can take this proctored exam and be eligible for credit."
"See your Progress page for a list of requirements in the order that they must be completed."